hr_holidays_imposed_days/models/hr_imposed_holidays.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+import math
+
+from openerp import models, fields, api, exceptions, _
+from openerp import tools
+
+
+class HrHolidaysImposed(models.Model):
+    _name = 'hr.holidays.imposed'
+    _description = 'Manage imposed holidays'
+
+    name = fields.Char(required=True)
```

Comment:
The string is automatically generated from the field's name.
It can be forgotten when you don't want to custimize it.